### PR TITLE
chore(ci): add commit linting workflow

### PR DESCRIPTION
**Pull request by Jayllyz**
Fix https://github.com/nodejs/amaro/issues/27

This action uses conventional commit config by default.

Successful run: https://github.com/nodejs/amaro/actions/runs/10213684789/job/28259584521?pr=31

### CODE REVIEW

File: .github/workflows/lint-pr.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Lint PR
```
Comment (AugustinMauroy) [inline .github/workflows/lint-pr.yml:1]: why not put it on CI.yml ?

Comment (Jayllyz) [reply .github/workflows/lint-pr.yml:1]: To make it work only on PR, but if necessary, I can move it to CI.

Comment (marco-ippolito) [reply .github/workflows/lint-pr.yml:1]: Yes please lets move it in CI

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -49,3 +49,13 @@ jobs:
       - run: npm install
       - run: npm run build
       - run: npm test
+
```
Comment (marco-ippolito) [inline .github/workflows/ci.yml:52]: we might want to run this before running tests

**Comment by marco-ippolito** [comment]
Can you please remove unreleated changes like empty new lines, so I can merge it

**Comment by Jayllyz** [comment]
> Can you please remove unreleated changes like empty new lines, so I can merge it.
> Also can you link a successful run?

Where should i link a successful run?

**Comment by marco-ippolito** [comment]
> > Can you please remove unreleated changes like empty new lines, so I can merge it.
> > Also can you link a successful run?
> 
> Where should i link a successful run?

your fork for example, so we can make sure its working before merging

**Review by marco-ippolito — APPROVED** [review]

**Pull request merged**